# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1656789374 -7200
# Parent  ba1725afff35b1cdcd37f0c1eebf462edeaceb4d
Feature: Multi-line tree follow-up: Style dummy/summary rows better.

diff --git a/mail/themes/linux/mail/messenger.css b/mail/themes/linux/mail/messenger.css
--- a/mail/themes/linux/mail/messenger.css
+++ b/mail/themes/linux/mail/messenger.css
@@ -505,8 +505,16 @@ treechildren::-moz-tree-cell(multiline) 
 }
 :root[uidensity="touch"] treechildren::-moz-tree-colspan {
   margin-top: 2.4rem;
 }
 :root[uidensity="touch"] treechildren::-moz-tree-cell(multiline) {
   min-height: 2.4rem;
   height: 2.4rem;
 }
+
+/* Special styling for the dummy/summary rows in grouped views */
+treechildren::-moz-tree-cell(multiline, dummy) {
+  height: 0 !important;
+}
+treechildren::-moz-tree-colspan(multiline, dummy) {
+  margin-top: 0 !important;
+}
diff --git a/mail/themes/osx/mail/messenger.css b/mail/themes/osx/mail/messenger.css
--- a/mail/themes/osx/mail/messenger.css
+++ b/mail/themes/osx/mail/messenger.css
@@ -494,8 +494,16 @@ treechildren::-moz-tree-cell(multiline) 
 }
 :root[uidensity="touch"] treechildren::-moz-tree-colspan {
   margin-top: 2.4rem;
 }
 :root[uidensity="touch"] treechildren::-moz-tree-cell(multiline) {
   min-height: 2.4rem;
   height: 2.4rem;
 }
+
+/* Special styling for the dummy/summary rows in grouped views */
+treechildren::-moz-tree-cell(multiline, dummy) {
+  height: 0 !important;
+}
+treechildren::-moz-tree-colspan(multiline, dummy) {
+  margin-top: 0 !important;
+}
diff --git a/mail/themes/windows/mail/messenger.css b/mail/themes/windows/mail/messenger.css
--- a/mail/themes/windows/mail/messenger.css
+++ b/mail/themes/windows/mail/messenger.css
@@ -578,8 +578,16 @@ treechildren::-moz-tree-cell(multiline) 
 }
 :root[uidensity="touch"] treechildren::-moz-tree-colspan {
   margin-top: 2.4rem;
 }
 :root[uidensity="touch"] treechildren::-moz-tree-cell(multiline) {
   min-height: 2.4rem;
   height: 2.4rem;
 }
+
+/* Special styling for the dummy/summary rows in grouped views */
+treechildren::-moz-tree-cell(multiline, dummy) {
+  height: 0 !important;
+}
+treechildren::-moz-tree-colspan(multiline, dummy) {
+  margin-top: 0 !important;
+}
